### PR TITLE
Ignore GR107

### PR DESCRIPTION
**Pull request by shmuel44**
## Related Issues
related: https://jira-dc.paloaltonetworks.com/browse/CIAC-11779

## Description

In the new GR107, we are converting all warnings in the old version to errors. 
This results in 235 errors when running with the -a flag.

To simplify the process for now, we will add an "ignore GR107" to the pack and open a [ticket](https://jira-dc.paloaltonetworks.com/browse/CIAC-11779) to address it in the future.

### CODE REVIEW

File: Packs/Carbon_Black_Enterprise_Response/.pack-ignore [hunk 1]
```diff
@@ -17,19 +17,32 @@ ignore=IN126
...
-ignore=BA108,BA109,BA124
+ignore=BA108,BA109,BA124,GR107
 
 [known_words]
 cb
 watchlists
-PREPREPRE
-POSTPOSTPOST
+preprepre
+postpostpost
```
Comment (dorschw) [inline Packs/Carbon_Black_Enterprise_Response/.pack-ignore:26]: intentional lowercasing?

Comment (shmuel44) [reply Packs/Carbon_Black_Enterprise_Response/.pack-ignore:26]: I don't remember this changes, i will revert this

Comment (shmuel44) [inline Packs/Carbon_Black_Enterprise_Response/.pack-ignore:26]:
```suggestion
PREPREPRE
POSTPOSTPOST
```

File: Packs/CommonPlaybooks/.pack-ignore [hunk 2]
```diff
@@ -1,28 +1,28 @@
 [known_words]
 xlsm
-Mimecast
+mimecast
```
Comment (shmuel44) [inline Packs/CommonPlaybooks/.pack-ignore:3]:
```suggestion
Mimecast
```

File: Packs/CommonPlaybooks/.pack-ignore [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
+sailpoint
+pingone
+clarizen
+exceedlms
+okta
+secneurx
+virustotal
+urlssl
+detectedhashes
+nonfoundhashes
```
Comment (shmuel44) [inline Packs/CommonPlaybooks/.pack-ignore:21]:
```suggestion
Zscaler
Symantec
Proofpoint
Stealthwatch
Sophos
Forcepoint
Netcraft
SailPoint
PingOne
Clarizen
ExceedLMS
Okta
SecneurX
VirusTotal
URLSSL
DetectedHashes
NonFoundHashes
```

File: Packs/CommonPlaybooks/.pack-ignore [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
 opswat
 filescan
-Stringify
-QR
+stringify
+qr
```
Comment (shmuel44) [inline Packs/CommonPlaybooks/.pack-ignore:25]:
```suggestion
Stringify
QR
```

File: Packs/CortexXDR/.pack-ignore [hunk 5]
```diff
@@ -54,10 +54,10 @@ xql
...
 setindicators
 printerrorentry
-SSO
-RDP
-XCLOUD
-Cryptomining
+sso
+rdp
+xcloud
+cryptomining
```
Comment (shmuel44) [inline Packs/CortexXDR/.pack-ignore:60]:
```suggestion
SSO
RDP
XCLOUD
Cryptomining
```

File: Packs/CortexXDR/.pack-ignore [hunk 6]
```diff
@@ -161,6 +161,11 @@ ignore=PB105
 [file:Cortex_XDR_-_Large_Upload.yml]
 ignore=PB121
 
-# GR103 is temporary, see CIAC-11705
 [file:Test_Playbook_-_Cortex_XDR_-_False_Positive_Incident_Handling.yml]
```
Comment (shmuel44) [inline Packs/CortexXDR/.pack-ignore:164]:
```suggestion
# GR103 is temporary, see CIAC-11705
[file:Test_Playbook_-_Cortex_XDR_-_False_Positive_Incident_Handling.yml]
```

File: Packs/Carbon_Black_Enterprise_Response/.pack-ignore [hunk 7]
```diff
@@ -26,10 +26,23 @@ PREPREPRE
 POSTPOSTPOST
 
 [file:CBFindIP.yml]
-ignore=BA124
+ignore=BA124,GR107
 
-# GR103 is temporary, see CIAC-11705
 [file:layoutscontainer-carbonblackEDR_alerts.json]
```
Comment (shmuel44) [inline Packs/Carbon_Black_Enterprise_Response/.pack-ignore:31]:
```suggestion
# GR103 is temporary, see CIAC-11705
[file:layoutscontainer-carbonblackEDR_alerts.json]
```

File: validation_config.toml [hunk 8]
```diff
@@ -72,7 +72,7 @@ select = [
     "IT102", "IT103",
     "MR100", "MR101", "MR106", "MR107", "MR108",
     "LO107",
-    "GR100", "GR104",
+    "GR100", "GR104","GR107",
```
Comment (shmuel44) [inline validation_config.toml:75]:
```suggestion
    "GR100", "GR104",
```

**Review by dorschw — APPROVED** [review]

**Pull request merged**